src/core/to-hash.js

```diff
@@ -8,8 +8,9 @@
  * @returns {String}
  */
 export let toHash = (str) => {
-    let i = 0,
+    let char,
+        i = 0,
         out = 11;
-    while (i < str.length) out = (101 * out + str.charCodeAt(i++)) >>> 0;
+    while ((char = str.charCodeAt(i++)) || char == char) out = (101 * out + char) >>> 0;
     return 'go' + out;
```

Comment:
With these changes the overall size increases 🤔 with `+5B`. Is there something that I am not checking correctly?